R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -3529,6 +3529,116 @@ test_that("gapply() and gapplyCollect() on a DataFrame", {
   })
 })
 
+test_that("gapply() Arrow optimization", {
+  skip_if_not_installed("arrow")
+  df <- createDataFrame(mtcars)
+
+  conf <- callJMethod(sparkSession, "conf")
+  arrowEnabled <- sparkR.conf("spark.sql.execution.arrow.enabled")[[1]]
+
+  callJMethod(conf, "set", "spark.sql.execution.arrow.enabled", "false")
```

Comment:
I am willing to make a wrapper for this `spark.sql.execution.arrow.enabled` set/unset but I would like to target this separately later if you guys won't mind.

Reply:
If this is mostly in tests I wouldn't bother with another method...